fetch non-public email and trust it.

revir opened the pull request.

revir committed:
fetch non-public email

revir committed:
set github's email varified

revir committed:
update

julianlam commented on README.md:
```diff
@@ -1,7 +1,13 @@
-    npm install nodebb-plugin-sso-github
\ No newline at end of file
+    npm install nodebb-plugin-sso-github2
+
```
README can be reverted 😄

julianlam commented on package.json:
```diff
@@ -16,24 +16,19 @@
+    "url": "https://github.com/revir/nodebb-plugin-sso-github2/issues"
   },
   "readme": "",
   "readmeFilename": "README.md",
```
This file can be reverted too, though feel free to keep your name in `contributors` 😃

julianlam commented on plugin.json:
```diff
@@ -1,8 +1,8 @@
+	"url": "https://github.com/revir/nodebb-plugin-sso-github2",
 	"library": "./library.js",
 	"hooks": [
 		{ "hook": "static:app.load", "method": "init" },
```
File can be reverted

julianlam reviewed (changes requested):
Looks good, though certain files need to be reverted to their original state. Thanks!

revir commented:
sorry, I fixed the two problems, and then I wanted to use it on my website, so I changed the readme and package.json, and published to npm, I thought it won't conflict with yours. 
Maybe you can just merge with those previous two changes, and then I will delete my repo, could you?
